services/essentials.yaml

```diff
@@ -10,8 +10,8 @@ spec:
       artifactId: configuration-as-code
       version: 0.11-alpha-rc362.942711740b07
     - groupId: io.jenkins.plugins
-      artifactId: essentials
-      version: 0.3-rc23.7230a0298a47
+      artifactId: evergreen
```

Comment:
Is it legal to have both of these plugins installed at once? I'm imagining what would happen with an existing instance when this update level is pushed out.

Reply:
Good point I overlooked. I _seem_ to remember some discussion around this, but cannot find it now. Also, nothing expressed about that in [BoM's JEP 309](https://github.com/jenkinsci/jep/tree/master/jep/309), I'll have a deeper look tomorrow I think.

We probably should have some way to actually say that a given plugin is *not* here for upgradeability purposes in the future IMO.

For now, I think I'm going to handle this by releasing an empty (aka _tombstoned_) essentials-plugin.

cc @oleg-nenashev for the BoM aspect, FYI.

Reply:
JEP-309 does not define behavior for Evergreen specifically. It's up to the BOM packaging logic to define and enforce behavior it needs, BOM just provides the documentation format. 

Tombstoning plugins and declaring dependencies on the new ones is what we actually do when there is an unavoidable need to change artifactId.

Reply:
@batmat the Update Manifest format has space for `upgrades` and `deletes`, we could look into using this as an opportunity to introduce the deletes :smile:

Reply:
Makes sense to me. See https://issues.jenkins-ci.org/browse/JENKINS-53164?focusedCommentId=347263&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-347263

Reply:
Tombstone release just done, through https://github.com/jenkinsci/evergreen-plugin/pull/14 and then released from my machine:
https://repo.jenkins-ci.org/releases/io/jenkins/plugins/essentials/0.4-essentials-tombstoned/